api/src/test/java/io/opentelemetry/metrics/CounterDoubleTest.java

```diff
@@ -70,6 +70,13 @@ public void preventNull_Unit() {
     meter.counterDoubleBuilder("metric").setUnit(null).build();
   }
 
+  @Test
+  public void preventNull_LabelKeys() {
```

Comment:
For better coverage, consider adding a test `preventNull_LabelKey`, i.e `setLabelKeys(Arrays.<String>asList(null))`. Same for other tests.

Reply:
Done.